### ISSUE TITLE
Create parallel approve flow with selected the person to approve

### ISSUE THREAD

**Issue by Saniwan** [MicrosoftDocs/power-automate-docs#262]
I already done to create parallel approve flow but I need to add more option to choose some person in parallel approve flow to approve not all person to approve. Only the person I selected the flow can send to them to approve. How can I create this flow?

**Comment by Dean-Haas** [comment]
Thanks for your feedback. For help with questions like this, ask your question on our forums: https://powerusers.microsoft.com/t5/Microsoft-Power-Automate/ct-p/MPACommunity
